pkg/kernel/networkservice/connectioncontextkernel/iptables4nattemplate/common.go

```diff
@@ -0,0 +1,121 @@
+// Copyright (c) 2022 Xored Software Inc and others.
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at:
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+//go:build linux
+// +build linux
+
+package iptables4nattemplate
+
+import (
+	"bytes"
+	"fmt"
+	"os"
+	"strings"
+
+	"github.com/edwarnicke/exechelper"
+	"github.com/pkg/errors"
+)
+
+type applyIPTablesKey struct {
+}
+
+type iptableManagerImpl struct {
+}
+
+// IPTablesManager provides methods for iptables nat rules management
+type IPTablesManager interface {
+	Get() (string, error)
+	Restore(string) error
+	Apply(string) error
+}
+
+func (m *iptableManagerImpl) Get() (string, error) {
+	cmdStr := "iptables-save"
+	buf := bytes.NewBuffer([]byte{})
+	err := exechelper.Run(cmdStr,
+		exechelper.WithStdout(buf),
+		exechelper.WithStderr(buf),
+	)
+	if err != nil {
+		err = errors.Wrapf(err, "%s", buf.String())
+		return "", err
+	}
+
+	return buf.String(), nil
+}
+
+func (m *iptableManagerImpl) Apply(rulesTemplate string) error {
+	rules := strings.Split(rulesTemplate, ";")
```

Comment:
```suggestion
	rules := strings.Split(rulesTemplate, "\n")
```

Reply:
`Apply` accepts `rules []string` now, spliting is no longer required.